doc/chips/10.rst

```diff
@@ -368,13 +369,50 @@ the initializer body:
      this.someOtherMethod(); // This call is valid, because we are in Phase 2
    }
 
+The compiler generated initializer will also include an argument-less
+``super.init()`` call after the initialization of its fields has been completed,
+meaning that if the parent type has defined an initializer with more than zero
+arguments, attempts to initialize the child will result in an error.  Parent
+types with a zero-argument defined initializer, or that rely on the compiler
+generated initializer, will resolve appropriately.  We may consider expanding to
+support the case where the parent has defined just a single initializer with
+any number of arguments, but that decision and its implementation are future
+work.
```

Comment:
I'm reading the diff and can't tell if there are a lot of other "We may..." sentences in here but it might be better to leave out.